resolwe/flow/executors/docker/run.py

```diff
@@ -0,0 +1,182 @@
+"""Docker workflow executor."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import json
+import os
+import shlex
+import subprocess
+import tempfile
+
+from ..local.run import FlowExecutor as LocalFlowExecutor
```

Comment:
The dots make sense in this particular case. Using absolute imports trips up the linter, because the executor isn't part of a resolwe package during runtime.